src/warning/warning-printer.ts

```diff
@@ -137,7 +137,7 @@ function getSquiggleUnderline(
     const endColumn = sourceRange.end.line === sourceRange.start.line ?
         sourceRange.end.column :
         lineText.length;
-    const prefix = ' '.repeat(startColumn);
+    const prefix = lineText.replace(/[^\t]/g, ' ').substr(0, startColumn);
```

Comment:
I think your code does the same thing as mine in a different way.  My patch was texted against a tab indented instance of:
```<img src="[[importPath]]/image.png">```

`importPath` was underlined due to https://github.com/Polymer/polymer-linter/issues/71.  This patch puts the underline in the correct place, the lineText.replace('/[^\t]/g', ' ') replaces all characters that are not tabs, the substr trims to the correct number of characters for the prefix.